src/main/java/com/internet/shop/controllers/ProductController.java

```diff
@@ -0,0 +1,25 @@
+package com.internet.shop.controllers;
+
+import com.internet.shop.lib.Injector;
+import com.internet.shop.model.Product;
+import com.internet.shop.service.interfaces.ProductService;
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class ProductController extends HttpServlet {
```

Comment:
We have a point in our checklist:
"Use informative names for your controllers. Always start controller’s name with verb: GetAllUsersController, AddProductToShoppingCartController etc."